Task-WebRTC.md

```diff
@@ -136,11 +138,15 @@ number.
 
 # Signalling Channel Handover
 
-As soon as both clients have exchanged the required messages and the
-WebRTC `RTCPeerConnection` instance informs the user application that
-the peer-to-peer connection setup is complete, the user application
-SHOULD request that the client hands over the signalling channel to a
-dedicated data channel:
+As soon as the user application has created a WebRTC `RTCPeerConnection`
+instance which intends to use this task for its signalling, the user
+application SHOULD request that the client hands over the signalling
+channel to a dedicated data channel before *offer* or *answer* of the
```

Comment:
Ah, I wasn't aware of that. My mental model was always that the datachannel is negotiated after the connection has been established.

If you're sure that this is the case, this is good to merge for me.